save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationVulnerabilitiesTab.kt

```diff
@@ -1,188 +0,0 @@
-                Fragment.create {
-                    td {
-                        +cellContext.value
-                    }
-                }
-            }
-            column(id = "progress", header = "Criticality", VulnerabilityMetadataDto::severityNum) { cellContext ->
-                Fragment.create {
-                    td {
-                        +cellContext.value.toString()
-                    }
-                }
-            }
-            column(id = "language", header = "Language", VulnerabilityMetadataDto::language) { cellContext ->
-                Fragment.create {
-                    td {
-                        +cellContext.value.toString()
-                    }
-                }
-            }
-        }
-    },
-    initialPageSize = 10,
-    useServerPaging = true,
-    tableOptionsCustomizer = ChildrenBuilder::enableExpanding
-) { props ->
-    arrayOf(props.count)
-}
-
-@Suppress("EMPTY_BLOCK_STRUCTURE_ERROR")
-val organizationVulnerabilitiesTab: FC<OrganizationVulnerabilitiesMenuProps> = FC { props ->
-    val filter = VulnerabilityFilter.approved.copy(
-        organizationName = props.organizationName,
-    )
-    val filterJson = Json.encodeToString(filter)
-
-    val (t) = useTranslation("organization")
-    var count by useState(initialValue = 0)
-    useRequest {
-        count = post(
-            url = "$vulnerabilitiesUrl/count/by-filter",
-            headers = jsonHeaders,
-            body = filterJson,
-            loadingHandler = ::noopLoadingHandler,
-            responseHandler = ::noopResponseHandler,
-        ).unsafeMap { response ->
-            response.decodeFromJsonString()
-        }
-    }
-    div {
-        className = ClassName("col-8 mx-auto mt-1 mb-3")
-
-        div {
-            className = ClassName("d-flex justify-content-center mb-3")
-
-            uploadCosvButton {
-                isImage = false
-            }
-        }
-        if (count != 0) {
-            vulnerabilityTable {
-                getData = { pageIndex, pageSize ->
-                    post(
-                        url = "$vulnerabilitiesUrl/by-filter",
-                        params = jso<dynamic> {
-                            page = pageIndex
-                            size = pageSize
-                        },
-                        headers = jsonHeaders,
-                        body = filterJson,
-                        loadingHandler = ::loadingHandler,
-                        responseHandler = ::noopResponseHandler,
-                    ).unsafeMap { response ->
-                        response.decodeFromJsonString()
-                    }
-                }
-                getPageCount = { pageSize ->
-                    pageCount(count, pageSize)
-                }
-                this.count = count
-            }
-        } else {
-            renderTablePlaceholder("text-center p-4 bg-white", "dashed") {
```

Comment:
I decided to remove this logic since we already have a message about no vulnerabilities in the table, and it fits better with the current implementation of the table with filters.